app/src/main/java/ru/noties/markwon/MarkdownRenderer.java

```diff
@@ -92,4 +99,25 @@ public void cancel() {
     private boolean isCancelled() {
         return task == null || task.isCancelled();
     }
+
+    private HeadingConfig getHeadingConfig() {
+        final HeadingTypeConfig h1 = new HeadingTypeConfig(-1, Color.RED, getTypeface(R.font.opensans_semibold));
```

Comment:
Should the hardcoded values RED and BLUE be there?

Reply:
This is just an example on how to use the config. It does not belong to the library